azurerm/resource_arm_monitor_metric_alert.go

```diff
@@ -37,16 +37,25 @@ func resourceArmMonitorMetricAlert() *schema.Resource {
 				ValidateFunc: validate.NoEmptyStrings,
 			},
 
+			"target_resource_type": {
```

Comment:
This looks unrelated to removing the max items property? PR title might warrant updating?